### PR TITLE
Small defender-for-identity/deploy/active-directory-federation-services corrections

### PR DESCRIPTION
**Pull request by bdeb1337**
Hello,

Hope you are doing well.

While working on the installation of Defender for Identity sensors on Active Directory Federation Services, I've noticed a small error in the connection string in the example given in this documentation: a semicolon is missing. This is added in the first commit.

image

Something else I've ran into, is that I was working with a group managed service account, as opposed to the example, which took me a while to remember that a `$` should be appended to the username. I've added a small hint in the documentation to remind the reader about this in the second commit of this PR.

Kind regards,

Bob

### CODE REVIEW

**Comment by Court72** [comment]
@batamig

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: ATPDocs/deploy/active-directory-federation-services.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ For example, the commands listed below might be helpful if you're using the Wind
 
 In these sample codes:
 
-- **[DOMAIN1\mdiSvc01]** is the directory services user of the workspace
+- **[DOMAIN1\mdiSvc01]** is the directory services user of the workspace (if you are working with a gMSA, use **[DOMAIN1\mdiSvc01$]** instead)
```
Comment (batamig) [inline ATPDocs/deploy/active-directory-federation-services.md:60]:
```suggestion
- **[DOMAIN1\mdiSvc01]** is the directory services user of the workspace. If you're working with a gMSA, append a **$** to the end of the username. For example: **[DOMAIN1\mdiSvc01$]**
```

**Comment by batamig** [comment]
Hi @bdeb1337, thanks for this PR! I've made a small suggestion to align your text with our style.
I'm also in the process of confirming the update with our team before we can merge the change.

File: ATPDocs/deploy/active-directory-federation-services.md (hunk 1 above)
Comment (bdeb1337) [reply ATPDocs/deploy/active-directory-federation-services.md]: Thank you, that's looking great.

**Pull request merged**